libretro/libretro.c

```diff
@@ -918,7 +859,7 @@ static void format_saved_memory(void)
 static void context_reset(void)
 {
     static bool first_init = true;
-    printf("context_reset.\n");
+    log_cb(RETRO_LOG_INFO, CORE_NAME ": context reset\n");
```

Comment:
Not like it matters much, but call it context_reset

Reply:
How about doing the right thing and using `RETRO_LOG_DEBUG` for these very low-level mesages and then use the function names?

Reply:
Sure